src/js/features/reducers/index.js

```diff
@@ -230,7 +232,22 @@ export default function reducer(state, action) {
         label: centerAlignGurbani ? 1 : 0,
       });
 
-      saveToLocalStorage(LOCAL_STORAGE_KEY_FOR_SPLIT_VIEW, centerAlignGurbani);
+      saveToLocalStorage(
+        LOCAL_STORAGE_KEY_FOR_CENTER_ALIGN_VIEW,
+        centerAlignGurbani
+      );
+      return {
+        ...state,
+        centerAlignGurbani,
+      };
+    }
+    case SET_CENTER_ALIGN_OPTION: {
+      const centerAlignGurbani = action.payload;
+
+      saveToLocalStorage(
+        LOCAL_STORAGE_KEY_FOR_CENTER_ALIGN_VIEW,
+        centerAlignGurbani
+      );
```

Comment:
What's the difference between toggle and set?

Reply:
Toggle - when user toggles the option. Set, used when user clicks reset. Followed the same pattern as used in `TOGGLE_DARK_MODE` `SET_DARK_MODE`

Reply:
@bogas04 is there any action needed here?